test/crd.go

```diff
@@ -162,6 +163,23 @@ func LatestService(namespace string, names ResourceNames, imagePath string) *v1a
 	}
 }
 
+// LatestServiceWithResources returns a RunLatest Service object in namespace with the name names.Service
+// that uses the image specified by imagePath.
```

Comment:
it seems to add constant resource requirements on top.

Reply:
Added to comment